.reuse/dep5

```diff
@@ -59,18 +59,18 @@ Copyright: 2013-2020 GitHub, Inc. and contributors <support@github.com>
 License: CC-BY-3.0
 
 Files: spdx-utils/src/main/resources/exceptions/*
-Copyright: 2011-2020 SPDX Workgroup <spdx-legal@lists.spdx.org>
+Copyright: © 2010-2020 Linux Foundation and its Contributors
```

Comment:
Isn't the specification licensed different than the actual license list? Even if not, according to your link this should be "CC-BY-3.0" then.

Reply:
https://github.com/spdx/spdx-spec is CC-BY-3.0, license list is CC0-1.0

Reply:
That's how I remembered it. However, I tried to find a source for the latter but failed. Do you have a source we could link to?

And: If the license list has a different license, doesn't it maybe also have a different copyright holder, namely the one we had before this change?

Finally: In any case, shouldn't we strip the `©` character?

Reply:
license list is CC0-1.0 as all data in a SPDX doc is CC0-1.0 per https://github.com/spdx/spdx-spec/blob/master/chapters/2-document-creation-information.md#22-data-license-

Reply:
I've added a link to the commit message and removed the special copyright character.

Reply:
We could strip © as it just symbol for copyright which is already clear from `copyright` variable name. Both  Copyright: `Copyright: © 2010-2020 Linux Foundation and its Contributors` and `Copyright: 2010-2020 Linux Foundation and its Contributors` are valid per  https://reuse.software/spec/

Reply:
> I've added a link to the commit message

Thanks, but could you please use a permalink using the `v3.9` tag for this?

Reply:
Done.

Reply:
I still haven't found a convincing source that the copyright holder should be the Linux Foundation instead of the SDPX Workgroup, but with the SDPX Workgroup being part of the Linux Foundation I *guess* this is fine.